docs/hammerdb.md

```diff
@@ -57,3 +57,49 @@ Once done creating/editing the resource file, you can run it by:
 # kubectl apply -f <path_to_cr_file>
 ```
 
+## Running hammerdb in VMs through kubevirt/cnv [Preview]
```

Comment:
In the table above it shows `Working` but here it has `[Preview]`

Reply:
Can you please be more specified which table you mean ?

Reply:
Fixed:
https://github.com/cloud-bulldozer/benchmark-operator/pull/569/commits/7d745e06ffa756531015344e5ecde4130329b17d